Gems/Atom/RPI/Code/Source/RPI.Public/Pass/RenderPass.cpp

```diff
@@ -47,6 +52,12 @@ namespace AZ
 
         const PipelineViewTag& RenderPass::GetPipelineViewTag() const
         {
+            if (m_viewTag.IsEmpty())
+            {
+                // This returns either the view-tag of the parent, the main ViewTag of the Pipeline, or a reference to a static but empty
+                // viewTag.
+                return Pass::GetPipelineViewTag();
+            }
             return m_viewTag;
         }
```

Comment:
I think you can just remove this function so it will just use Pass::GetPipelineViewTag();

Reply:
removed